src/languages/yaml.js

```diff
@@ -20,12 +20,12 @@ export default function(hljs) {
   const KEY = {
     className: 'attr',
     variants: [
-      // added brackets support 
-      { begin: /\w[\w :()\./-]*:(?=[ \t]|$)/ },
+      // added brackets support
+      { begin: /\w[\w :()\./\-*@]*:(?=[ \t]|$)/ },
       { // double quoted keys - with brackets
-        begin: /"\w[\w :()\./-]*":(?=[ \t]|$)/ },
+        begin: /"\w[\w :()\./\-*@]*":(?=[ \t]|$)/ },
       { // single quoted keys - with brackets
-        begin: /'\w[\w :()\./-]*':(?=[ \t]|$)/ },
+        begin: /'\w[\w :()\./\-*@]*':(?=[ \t]|$)/ },
```

Comment:
Can we please keep the `-` at the very end of the character group so we don't need the extra escape?

Also, could we get a test case added to the markup tests to show that this works as intended?

Reply:
hello! @joshgoebel. Can you please review this part?